src/lib/merkle_mask/maskable_merkle_tree.ml

```diff
@@ -142,14 +145,44 @@ module Make (Inputs : Inputs_intf) = struct
     Uuid.Table.add_multi registered_masks ~key:(get_uuid t) ~data:attached_mask ;
     attached_mask
 
-  let unregister_mask_exn (t : t) (mask : Mask.Attached.t) =
+  (* TODO We have the parent already from mask.parent. remove extra param. *)
```

Comment:
Does this TODO still need to be addressed before this PR lands? If not, either remove it if it isn't necessary anymore, or open and issue for it and add the number to this comment.